src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/config/ManageUsersApiExceptionHandler.kt

```diff
@@ -161,6 +162,20 @@ class HmppsManageUsersApiExceptionHandler {
       )
   }
 
+  @ExceptionHandler(GroupNotFoundException::class)
+  fun handleGroupNotFoundException(e: GroupNotFoundException): ResponseEntity<ErrorResponse> {
+    log.debug("Username not found exception caught: {}", e.message)
+    return ResponseEntity
+      .status(NOT_FOUND)
```

Comment:
May need to add `.contentType(APPLICATION_JSON)` if you force the exception in postman the type will be wrong

Reply:
Added